viewer/packages/360-images/src/icons/IconCollection.ts

```diff
@@ -85,6 +125,29 @@ export class IconCollection {
     };
   }
 
+  private computeProximityPoints(octree: IconOctree, iconSprites: InstancedIconSprite): BeforeSceneRenderedDelegate {
+    return ({ camera }) => {
+      const points =
+        this._proximityRadius < 1
+          ? this._icons
+          : octree.findPoints(camera.position, this._proximityRadius).map(pointContainer => {
+              return pointContainer.data;
+            });
+
+      let closestPoints = points.sort((a, b) => {
+        return b.position.distanceTo(camera.position) - a.position.distanceTo(camera.position);
```

Comment:
Tiny optimization, but it is more efficient to check the "squared distance to", which is preferable to comparing vector lengths. See: https://threejs.org/docs/#api/en/math/Vector3.distanceToSquared

Reply:
Done! 🥳